builds/jazz-build-module/aws-lambda-events-module.groovy

```diff
@@ -240,29 +240,48 @@ def checkAndConvertEvents(events){
 
 def getStreamEnabledArn(tableStreamArn) {
   def tableName = tableStreamArn.tokenize("/").last()
-  def streamList = sh(
-    script: "aws dynamodbstreams list-streams --table-name ${tableName} --region ${config_loader.AWS.REGION} --output json",
-    returnStdout: true
-  ).trim()
-  def streamListJson = parseJson(streamList)
+  try {
+    sh "aws dynamodb describe-table --table-name ${tableName} --region ${config_loader.AWS.REGION} --output json"
+    echo "table exist..."
+    def streamList = sh(
+      script: "aws dynamodbstreams list-streams --table-name ${tableName} --region ${config_loader.AWS.REGION} --output json",
+      returnStdout: true
+    ).trim()
+    def streamListJson = parseJson(streamList)
 
-  if (streamListJson.Streams.size() == 0) {
-    return createDynamodbStream(tableName)
-  } else {
-    def streamArnList = streamListJson.Streams
-    for (stream in streamArnList) {
-      def streamDetails = sh(
-        script: "aws dynamodbstreams describe-stream --stream-arn ${stream.StreamArn} --region ${config_loader.AWS.REGION} --output json",
-        returnStdout: true
-      ).trim()
-      def streamDetailsJson = parseJson(streamDetails)
+    if (streamListJson.Streams.size() == 0) {
+      return createDynamodbStream(tableName)
+    } else {
+      def streamArnList = streamListJson.Streams
+      for (stream in streamArnList) {
+        def streamDetails = sh(
+          script: "aws dynamodbstreams describe-stream --stream-arn ${stream.StreamArn} --region ${config_loader.AWS.REGION} --output json",
+          returnStdout: true
+        ).trim()
+        def streamDetailsJson = parseJson(streamDetails)
 
-      if ((streamDetailsJson.StreamDescription.StreamStatus == "ENABLED") || (streamDetailsJson.StreamDescription.StreamStatus == "ENABLING")) {
-        return stream.StreamArn
-      } else if (streamArnList.last().StreamArn == stream.StreamArn) {
-        return createDynamodbStream(tableName)
+        if ((streamDetailsJson.StreamDescription.StreamStatus == "ENABLED") || (streamDetailsJson.StreamDescription.StreamStatus == "ENABLING")) {
+          return stream.StreamArn
+        } else if (streamArnList.last().StreamArn == stream.StreamArn) {
+          return createDynamodbStream(tableName)
+        }
       }
     }
+  } catch (ex) {
+    def response
+    try {
+      response = sh(
+        script: "aws dynamodb describe-table --table-name ${tableName} --region ${config_loader.AWS.REGION} --output json 2<&1 | grep -c 'ResourceNotFoundException'",
+        returnStdout: true
+      ).trim()
+    } catch (e) {
+      echo "Error occured while describing the dynamodb"
+    }
+    if (response) {
+      echo "Dynamodb does not exists"
```

Comment:
Dynamodb table does not exist (maybe even provide the name of the table that was the input

Reply:
Included table name